molo/yourwords/models.py

```diff
@@ -0,0 +1,26 @@
+from wagtail.wagtailcore.models import Page
+
+
+class Competition(Page):
+    name = Page.CharField(max_length=128)
+    description = Page.TextField()
+    content = Page.TextField()
+    start_date = Page.DateTimeField(null=True, blank=True)
+    end_date = Page.DateTimeField(null=True, blank=True)
+
+
+class CompetitionEntry(Page):
+    story_name = Page.CharField(max_length=128)
+    story_text = Page.TextField()
+    TERMS_OR_CONDITIONS_CHOICES = (
```

Comment:
You don't need these choices anymore - a boolean field is either True/False. (same for the hide real name below)